src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,79 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Create" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+
+        @ignore
+        Scenario: Verify new resource can be created successfully
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  | setting          | value                                          |
+                  | codeValue        | Sick Leave                                     |
+                  | description      | Sick Leave                                     |
+                  | namespace        | uri://ed-fi.org/AbsenceEventCategoryDescriptor |
+                  | shortDescription | Sick Leave                                     |
+             Then it should respond with 201
+              And the record can be retrieved with a GET request
+              # TODO: when writing the step definition,
+              # be sure to validate that the response matches the original payload,
+              # except for the addition of the 'id' field
+
+        @ignore
+        Scenario: Verify error handling with POST using invalid data
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  | setting          | value                                          |
+                  | codeValue        |                                                |
+                  | description      | Wrong Value                                    |
+                  | namespace        | uri://ed-fi.org/AbsenceEventCategoryDescriptor |
+                  | shortDescription | Wrong Value                                    |
+             Then it should respond with 400
+              And the response message includes:
+                  | Validation of 'AbsenceEventCategoryDescriptor' failed.\r\n\tCodeValue is required.\n |
```

Comment:
For responses, can you provide the exact expected JSON? Copying from the ODS/API 7.2, but replacing the actual correlationId with `null`.

Reply:
Error messages were updated using ODS/API 7.2